### PR TITLE
Basic UI

### PR DESCRIPTION
**Pull request by hopeforkoami**
In this branch , I :
 - Made sure my console app behave in the same way as it is in the example.
 - Create a` app.rb` file that will serve as your console app `entry-point`. It has methods that do the following:
    * List all books.
    * List all people.
    * Create a person (teacher or student, not a plain Person).
    * Create a book.
    * Create a rental.
    * List all rentals for a given person id.

 - In my `main.rb`, I define the entry point, which is a method called main that is invoked at the end of my file. This method is doing the following:
    * Present the user with a list of options to perform.
    * Lets users choose an option.
    * If needed, ask for parameters for the option.
    * Have a way to quit the app.

Thanks

### CODE REVIEW

File: entity/student.rb [hunk 1]
```diff
@@ -16,4 +16,8 @@ def classroom=(class_room)
     @classroom = class_room
     class_room.students.push(self) unless class_room.students.include?(self)
   end
+
+  def display_person
+    '[Student]  Name: '.concat(@name).concat(' , ID: ').concat(@id.to_i).concat(' , age: ').concat(@age)
+  end
```
Comment (AnselemOdims) [inline entity/student.rb]: - [ ] At the moment, when a new student is created and listed the ID of the student does not show which in turn makes it impossible to create a rental for that student. Please kindly fix it.

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @hopeforkoami , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: entity/student.rb (hunk 1 above)
Comment (Whoistolu) [reply entity/student.rb]: - Well resolved 👍🏽 
I have been able to create Rental successfully!
Screenshot 2023-02-11 at 23 22 02

**Review by Whoistolu — APPROVED** [review]
### Hi @hopeforkoami 

Great job implementing the recommended changes by the previous reviewer 👍🏽 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**